server/server.go

```diff
@@ -183,3 +200,9 @@ func IsBlacklisted(ip string) bool {
 	}
 	return false
 }
+
+func setHeader(respw http.ResponseWriter) http.ResponseWriter {
```

Comment:
does not need to return the writer, and should probably be called `setCorsHeaders`

Reply:
updated